packages/cli/src/commands/dbCommands/__tests__/dbCommands.test.js

```diff
@@ -33,29 +35,29 @@ describe('db commands', () => {
   })
 
   it('runs the command as expected', async () => {
-    await up.handler({ dbClient: true })
+    await up.handler({ dbClient: true, schema })
```

Comment:
Maybe we can have a test that doesn't path in the schema, so that we can get the default value?

Or was the test breaking since it couldn't find `redwood.toml`?

Reply:
Ok, I'll show you how to work around that tomorrow 👍 , it's a bit late here.

Reply:
If the schema variable isn't passed to the handler here it fails.
I have tried to make the schema optional with:
```
schema && `--schema=${schema}`
```
But this was causing other issues. 
Overall, the way it is currently implemented requires the schema to be passed to the handler. This schema can be either configured by the user in redwood.toml, in which case this custom value is passed, otherwise if not configured, the default value is passed, in our case `./api/db/schema.prisma`.